funnel/models/event.py

```diff
@@ -125,6 +134,24 @@ class Participant(BaseMixin, db.Model):
     __table_args__ = (db.UniqueConstraint('proposal_space_id', 'email'),)
 
     @classmethod
+    def get(cls, space, email, participant_attrs={}, create=False):
+        participant = cls.query.filter_by(proposal_space=space, email=email).first()
```

Comment:
This assumes `email` is unique. How does this work with platforms like Explara and DoAttend that don't enforce a unique constraint on email?